packages/react-dom/src/__tests__/ReactStatelessComponent-test.js

```diff
@@ -149,10 +149,11 @@ describe('ReactStatelessComponent', () => {
     }).toThrowError(
       __DEV__
         ? 'Stateless function components cannot have refs.'
-        : // TODO: the different message in production seems like a bug.
-          // It happens because we don't save _owner in production for
-          // functional components. We should probably show a better message.
-          'Element ref was specified as a string (me) but no owner was set.',
+        : // It happens because we don't save _owner in production for
+          // functional components.
+          'Element ref was specified as a string (me) but no owner was set. ' +
+          "You may be adding a ref to a component that was not created inside a component's render method, or " +
```

Comment:
I think this message doesn't make it clear that this could be due to using a string ref in a functional component.
Can we split it in three parts? e.g.

```
Element ref was specified as a string (me) but no owner was set. This could happen for one of the following reasons:

1.
2.
3.
```

Reply:
Of course. I will update it accordingly.